src/components/geojson-map.vue

```diff
@@ -196,6 +210,11 @@ const fitView = (extent, options = undefined) => {
   });
 };
 
```

Comment:
> It makes sense that fitToAllFeatures() is its own function given that it's called by an event handler in the template. However, I'm wondering if there's a way to increase code reuse...

Main place map is changed to show different data:
- initial map load in `show()` - show all features, don't animate
- button click - show all features, DO animate
- cluster click with fewer than 1000 data points - show all cluster features, DO animate
- cluster click with more than 1000 data points - center on cluster and zoom in, DO animate

Every case is a little different so I ended up keeping `fitViewToAllFeatures()` to fit to everything, but using it in `show`().  I didn't want `animate` as the very first argument to `fitView()` since it felt too specific for an otherwise general function.